### PR TITLE
Add COBLET 2022 post

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

**Comment by gallardoalba** [comment]
Related: https://github.com/usegalaxy-eu/website/issues/1065#issuecomment-1318264987

File: src/blog/2022-10-19-COBLET-talk/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+date: '2020-01-07'
```
Comment (dannon) [inline src/blog/2022-10-19-COBLET-talk/index.md:2]: Looks like this needs a tweak?

Comment (gallardoalba) [reply src/blog/2022-10-19-COBLET-talk/index.md:2]: Ah sorry, the date is wrong.

Comment (gallardoalba) [inline src/blog/2022-10-19-COBLET-talk/index.md:2]:
```suggestion
date: '2022-10-14'
```

**Comment by bgruening** [comment]
Thanks for moving this over!

**Comment by gallardoalba** [comment]
> @gallardoalba we can and should add more tags to such posts. E.g. the `tags` and subsites.

Ah ok, I didn't realize that it was possible.

File: src/blog/2022-10-19-COBLET-talk/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+date: '2022-10-14'
+title: 'Galaxy in COBLET2022'
+authors: "Cristóbal Gallardo"
+tease: "Cristóbal presented the Galaxy Project-VGP collaboration about Genome Assembly at COBLET2022"
+tags: [vgp,erga]
```
Comment (gallardoalba) [inline src/blog/2022-10-19-COBLET-talk/index.md:6]:
```suggestion
tags: [vgp,erga,education,teaching]
```

**Comment by gallardoalba** [comment]
Thanks! Indeed it was a collaborative post at the end :dancers:

**Pull request merged**